stable/rook-ceph/templates/NOTES.txt

```diff
@@ -0,0 +1,20 @@
+The Rook Operator has been installed. Check its status by running:
+  kubectl --namespace {{ .Release.Namespace }} get pods -l "app=rook-ceph-operator"
+
+Visit https://rook.io/docs/rook/master for instructions on how to create and configure Rook clusters
+
+Note: You cannot just create a CephCluster resource, you need to also create a namespace and
+install suitable RBAC roles and role bindings for the cluster. The Rook Operator will not do
+this for you. Sample CephCluster manifest templates that include RBAC resources are available:
+
+- https://rook.github.io/docs/rook/master/ceph-quickstart.html
+- https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/cluster.yaml
```

Comment:
would be nice to have more direct links other than "master" here, like the concrete version we're supporting.